wcfsetup/install/files/style/ui/discussionList.scss

```diff
@@ -0,0 +1,303 @@
+.discussionList {
+	display: flex;
+	flex-direction: column;
+}
+
+.discussionList__item {
+	display: flex;
+	flex-direction: column;
+	gap: 10px;
+	position: relative;
+}
+
+.discussionList__item:not(:last-child) {
+	border-bottom: 1px solid var(--wcfContentBorderInner);
+}
+
+@include screen-sm-down {
+	.discussionList__item:not(:last-child) {
+		padding-bottom: 15px;
+		margin-bottom: 15px;
+	}
+}
+
+@include screen-md-up {
+	.discussionList__item:not(:last-child) {
+		padding-bottom: 20px;
+		margin-bottom: 20px;
+	}
+}
+
+.discussionList__item__header {
+	align-items: center;
+	display: flex;
+	gap: 10px;
+	justify-content: space-between;
+}
+
+.discussionList__item__meta {
+	align-items: center;
+	display: flex;
+	gap: 10px;
+
+	color: var(--wcfContentDimmedText);
+
+	@include wcfFontSmall;
+
+	.icon {
+		color: inherit;
+	}
```

Comment:
```suggestion
```

We do not use the `icon` class anymore. Besides that the icons will always inherit the text color since 6.0.